### PR TITLE
Add Age Clock

### PR DESCRIPTION
**Pull request by nomasprime**

### CODE REVIEW

File: Time/age-clock.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+# <xbar.author>Richard Jones</xbar.author>
+# <xbar.author.github>nomasprime</xbar.author.github>
+# <xbar.desc>Displays your age in a clock format [y:m:d]</xbar.desc>
+# <xbar.dependencies>python</xbar.dependencies>
+
+from dateutil.relativedelta import *
+from datetime import datetime
+
+today = datetime.today()
+dob = datetime(1979, 11, 8)
```
Comment (sprak3000) [inline Time/age-clock.py:14]: Consider using [xbar variables](https://xbarapp.com/docs/2021/03/14/variables-in-xbar.html) to allow others to set their own birthday.

File: Time/age-clock.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python3
```
Comment (sprak3000) [inline Time/age-clock.py:1]: This change allows python to be picked up from a variety of places -- macOS install, Homebrew on Intel silicon install, Homebrew on M1 silicon install, etc.

```suggestion
#!/usr/bin/env -S PATH="${PATH}:/opt/homebrew/bin:/usr/local/bin" python3
```

**Review by sprak3000 — APPROVED** [review]
Nice! Thank you for your contribution!

**Comment by nomasprime** [comment]
My pleasure, always happy to contribute.

**Pull request merged**